content/en-us/reference/engine/globals/LuaGlobals.yaml

```diff
@@ -725,3 +728,28 @@ functions:
         summary: ''
     tags:
     code_samples:
+  - name: ypcall
+    summary: |
+      Similar to `pcall()` expect it yields.
```

Comment:
This is incorrect, `ypcall` doesn't yield by itself. In fact, it is absolutely identical to `pcall`.